src/ray/core_worker/core_worker.cc

```diff
@@ -616,6 +617,11 @@ void CoreWorker::OnNodeRemoved(const rpc::GcsNodeInfo &node_info) {
   }
 }
 
+void CoreWorker::OnObjectStoreFull(size_t bytes_to_allocate) {
+
+  TriggerGlobalGC();
```

Comment:
We shouldn't need this right? In the final implementation, spilling is triggered in raylet only during a Create() or Get() call, never from core worker.

Reply:
Sure. But that could only be done after some refactoring, because currently the Create() call directly goes into plasma store, and plasma store has no access to raylet members.

Reply:
Still, let's not add code that won't be relevant for the final design. Please remove this.